src/gdcmodels/esmodels/segment_cnv_centric/mapping.yaml

```diff
@@ -0,0 +1,606 @@
+                type: keyword
+              secondary_gleason_grade:
+                normalizer: clinical_normalizer
+                type: keyword
+              site_of_resection_or_biopsy:
+                normalizer: clinical_normalizer
+                type: keyword
+              state:
+                normalizer: clinical_normalizer
+                type: keyword
+              submitter_id:
+                type: keyword
+              synchronous_malignancy:
+                normalizer: clinical_normalizer
+                type: keyword
+              tissue_or_organ_of_origin:
+                normalizer: clinical_normalizer
+                type: keyword
+              treatments:
+                properties:
+                  chemo_concurrent_to_radiation:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  days_to_treatment_end:
+                    type: long
+                  days_to_treatment_start:
+                    type: long
+                  initial_disease_status:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  number_of_cycles:
+                    type: long
+                  regimen_or_line_of_therapy:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  state:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  submitter_id:
+                    type: keyword
+                  therapeutic_agents:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  treatment_anatomic_site:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  treatment_dose:
+                    type: long
+                  treatment_frequency:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  treatment_id:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  treatment_intent_type:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  treatment_or_therapy:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  treatment_outcome:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                  treatment_type:
+                    normalizer: clinical_normalizer
+                    type: keyword
+                type: nested
+              tumor_grade:
+                normalizer: clinical_normalizer
+                type: keyword
+              year_of_diagnosis:
+                type: long
+            type: nested
+          disease_type:
+            normalizer: clinical_normalizer
+            type: keyword
+          exposures:
```

Comment:
The `cnv_occurrence_centric` mapping includes `bmi` under exposures. Since the ticket specifies this should be identical, should it either be added here or perhaps removed from `cnv_occurrence_centric`?

Reply:
oh i made a mistake in the jira ticket. it should say identical to the occurrence object in cnv_centric.

Reply:
https://gdc-ctds.atlassian.net/browse/DICT-170

so bmi under exposure is being deprecated, so I'm thinking 1) we don't need it for this mapping under exposures 2) it probably should be removed in cnv_occurrence_centric

Reply:
Ah, that's my bad, I misread. Thank you for the clarification. And agreed, no need to include bmi here. We should probably remove bmi from cnv_occurrence_centric but that's out of scope for this effort.